tools/querytee/proxy.go

```diff
@@ -46,6 +49,12 @@ func (cfg *ProxyConfig) RegisterFlags(f *flag.FlagSet) {
 	f.BoolVar(&cfg.UseRelativeError, "proxy.compare-use-relative-error", false, "Use relative error tolerance when comparing floating point values.")
 	f.DurationVar(&cfg.SkipRecentSamples, "proxy.compare-skip-recent-samples", 60*time.Second, "The window from now to skip comparing samples. 0 to disable.")
 	f.BoolVar(&cfg.PassThroughNonRegisteredRoutes, "proxy.passthrough-non-registered-routes", false, "Passthrough requests for non-registered routes to preferred backend.")
+	f.Func("backend.filter", "A request filter as a regular expression. Only matches are proxied to non-preferred backends.", func(raw string) error{
+		var err error
+		cfg.RequestURLFilter, err = regexp.Compile(raw)	
+		return err
+	})
+	f.BoolVar(&cfg.InstrumentCompares, "proxy.compare-instrument", false, "Reports metrics on comparisons of responses between preferred and secondary endpoints for supported routes.")
```

Comment:
Are "secondary endpoints" and "non-preferred backends" the same thing?

Reply:
Yes. I'll correct the help string.

Reply:
Done :slightly_smiling_face: